Menambahkan kode untuk menghandle akun dengan > 1 role/kode_org

imiro opened the pull request:
Solusi sederhana untuk issue #10 
Mungkin perlu dipikirkan adanya kemungkinan bahwa bukan hanya mahasiswa yang dapat memiliki >1 role?

imiro committed:
Menambahkan kode untuk menghandle akun dengan > 1 role

albadrun commented:
oke, di-review dulu ya, sorry agak lama, hehe

fatanugraha commented on SSO/SSO.php:
```diff
@@ -127,10 +127,22 @@ public static function getUser() {
+	      $user->study_program = $data['study_program'];
+	      $user->educational_program = $data['educational_program'];
+      } else {
+	      $user->faculty = array();
```
We might want to keep this property's type consistent to make sure existing apps that use this library did not break.

One that i can suggest is keep `faculty`, `study_program`, and `educational_program` to contain the first role, and put the others to another property.

albadrun replied:
agree with this

fatanugraha reviewed (changes requested):
might want to wait for other mantainer's input

imiro committed:
Give additional properties to user object

When more than 1  `kd_org` is present for a single user, 
additional faculty, study_program, and educational_program are 
stored in properties named after their plural forms, as arrays.

This is done to ensure type consistency of existing properties.

imiro commented:
Hi,

Sorry for letting this off for months. Thank you for the feedback, added commit 13667ee to address this issue.

imiro committed:
Merge update from base repository

Add Sub Specialist Program FK (#9)
